### DISCUSSION TITLE
A question about scientific research

### DISCUSSION THREAD

**Discussion by finger1517** [facebookresearch/fastMRI#200] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/33984530/147231654-5201f137-ae86-4f83-8284-14671920d504.png)
i-RIM method

![image](https://user-images.githubusercontent.com/33984530/147231816-10422dc9-2807-4620-92e6-91d329c9853f.png)
joint-IC method

in some of the fastMRI challenge research teams they published the paper and used the test set to check the reconstruction quality of their methods, I wonder if they do not have the ground truth image, how they test their reconstruction images?

what is more, if later paper wants to do the research and compare the results with other works, how they compare with each other? using validation set? or other choice are available?

Could you answer my confusion?
thank you,
Ayuan yang.

**Comment by mmuckley** [comment, marked as answer]
Converting to a discussion as this does not document a bug.

@finger1517, at the moment the only way to validate on the entire test set is to use the public website at fastmri.org. You can upload your reconstructions to the site and the system will score them for you at 4X and 8X accelerations.

If you want to run evaluations at different accelerations, at the moment the only thing you can do is to re-split the validation set.

**Reply by finger1517** [reply]
Got the details! Thanks for your reply!
